cairo/ethereum/cancun/transactions.cairo

```diff
@@ -1,122 +1,162 @@
-from ethereum_types.bytes import BytesStruct, Bytes, Bytes0, TupleBytes32
-from ethereum_types.numeric import Uint, U256, U64
-from ethereum.cancun.fork_types import Address, TupleVersionedHash
-
-const TX_BASE_COST = 21000;
```

Comment:
tx types being in transactions.py i wouldn't have moved them out (unless there's another reason)